features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -232,6 +232,13 @@
 		<bundle>mvn:org.eclipse.kura/org.eclipse.soda.dk.comm.x86_64/1.2.201</bundle>
 	</feature>
 
+	<feature name="openhab.tp-serial-purejavacomm" version="${project.version}">
+		<capability>openhab.tp;feature=serial;impl=purejavacomm</capability>
+		<bundle dependency="true">mvn:net.java.dev.jna/jna-platform/5.6.0</bundle>
```

Comment:
Not sure if you wish to embed jna second time. As far I remember there is `openhab.tp-jna` feature which couls be utilized if you align jna version in pom.xml.

Reply:
Indeed, I was not aware jna was already in a specific dependency.